src/algebra/archimedean.lean

```diff
@@ -26,11 +26,13 @@ variables [linear_ordered_ring α] [archimedean α]
 
 lemma pow_unbounded_of_one_lt (x : α) {y : α}
     (hy1 : 1 < y) : ∃ n : ℕ, x < y ^ n :=
-have hy0 : 0 <  y - 1 := sub_pos_of_lt hy1,
+have hy0 : 0 < y - 1 := sub_pos_of_lt hy1,
+-- TODO `by linarith` fails to prove hy1'
```

Comment:
This is because `ring` expects a commutative ring, and `linarith` depends on `ring`.